monkey/infection_monkey/master/exploiter.py

```diff
@@ -55,7 +56,7 @@ def exploit_hosts(
         current_depth: int,
         servers: Sequence[str],
         results_callback: Callback,
-        scan_completed: Event,
+        scan_completed: ThreadingEvent,
```

Comment:
I think it's better to just `import threading` and then use `threading.Event` for the type hints in this file. It's 1 more character, but no one needs to look up what it is.